examples/va/hci/edpm-pre-ceph/values.yaml

```diff
@@ -103,6 +103,16 @@ data:
         storage_mgmt_vlan_id: 23
         storage_mgmt_cidr: "24"
         storage_mgmt_host_routes: []
+    networks:
```

Comment:
This is not required as you've networks in the nodes section, you've to fix the references of this in kustomize.

Reply:
Looks like we assume that section to be there in every spec https://github.com/openstack-k8s-operators/architecture/blob/main/lib/dataplane/kustomization.yaml#L24-L34 which is incorrect.  `nodeTemplate`  `networks` are only used if not provided in the `node` section.